packages/payload/src/queues/config/jobsCollection.ts

```diff
@@ -119,6 +119,21 @@ export const getDefaultJobsCollection: (config: Config) => CollectionConfig | nu
                     name: 'output',
                     type: 'json',
                   },
+                  {
```

Comment:
This is going to be a breaking change which is why I'm very cautious about getting this merged.